alamo-classes/2021FALL-GAME2332
# Gravity movement

Closes #48 

Test Instructions:
     1. Open the "PlanetGravity" Scene and hit the play button
     2. Move the mouse horizontally side to side
           The camera should follow the mouse movement side to side
     3. Move the mouse vertically up and down
            The camera should go up and down following the mouse movement
     4. Use WASD to get to the underside of the planet ( Can sprint and jump there too )
            Player should move as normal and the camera should follow the player's orientation
     5. Repeat steps 2 and 3
